lib/tasks/stop_data_export.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'csv'
+namespace :bus_stops do
+  task export: :environment do
+    CSV.open('old_stop_data.csv', 'w', write_headers: true,
```

Comment:
What does `write_headers: true` do?

Reply:
Also, why allow for specifying the CSV name in one task, but hardcode it in the other?